### PR TITLE
Brexit Secondary Results component

### PR DESCRIPTION
**Pull request by sumeetadur**
Using the Brexit poll-of-polls graphic within the secondary Brexit component:

screen shot 2016-06-02 at 11 08 34

### CODE REVIEW

**Comment by sumeetadur** [comment]
LGTM 👍

**Comment by sumeetadur** [comment]
@callumlocke could you have a look?

File: shared/components/brexit-secondary-results/brexit-secondary-results.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+		return `${daystogo} days until the referendum`;
+	}
+};
+
+export default props => <div className="brexit-secondary-results card" data-size="medium">
+							<div className="brexit-secondary-results__content">
+								<h3 className="brexit-secondary-results__title card__title">{getCountdownText()}</h3>
+								{props.data.brexitSecondaryResults.map( poll => <Section raw={`<a class="brexit-secondary-results__chart brexit-secondary-results__chart--${poll.size}" href="https://ig.ft.com/sites/brexit-polling/">${poll.svg}</a>`} /> )}
+							</div>
+						</div>;
```
Comment (callumlocke) [inline shared/components/brexit-secondary-results/brexit-secondary-results.js]: can you format this bit like this:

```js
export default props => (
  <div className="brexit-secondary-results card" data-size="medium">
    <div className="brexit-secondary-results__content">
      <h3 className="brexit-secondary-results__title card__title">{getCountdownText()}</h3>
      {props.data.brexitSecondaryResults.map( poll => <Section raw={`<a class="brexit-secondary-results__chart brexit-secondary-results__chart--${poll.size}" href="https://ig.ft.com/sites/brexit-polling/">${poll.svg}</a>`} /> )}
    </div>
  </div>
);
```

File: server/libs/brexit-secondary-results.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+		height: 75
+	}
+];
+
+const widthHeightArray = [];
+for( const { size, width, height } of charts ) {
+	widthHeightArray.push(`${width}-x-${height}`);
+}
+
+const poller = new Poller({url: `https://ig.ft.com/sites/brexit-polling/poll-of-polls/multiple/${widthHeightArray.join('_')}.json`})
```
Comment (callumlocke) [inline server/libs/brexit-secondary-results.js]: in my opinion lines 22–27 would be simpler with `.map()`:

```js
const sizes = charts.map(({ width, height }) => `${width}-x-${height}`).join('_');

const poller = new Poller({url: `https://ig.ft.com/sites/brexit-polling/poll-of-polls/multiple/${sizes}.json`})
```

**Comment by callumlocke** [comment]
👍  this looks good

Please find a Next team member to check it over before merging.

**Comment by sumeetadur** [comment]
@ironsidevsquincy could you have a look?

File: shared/components/brexit-secondary-results/_brexit-secondary-results.scss [hunk 3]
```diff
@@ -0,0 +1,65 @@
+$button-color-darker: oColorsGetPaletteColor('pink-tint2');
+$button-color-darker-still: darken($button-color-darker, 3%);
+
+.brexit-secondary-results {
+	text-align: center;
+
+	&__content {
```
Comment (ironsidevsquincy) [inline shared/components/brexit-secondary-results/_brexit-secondary-results.scss]: this pattern makes it tricky to grep. does it need to be nested within `.brexit-secondary-results` at all?

**Comment by ironsidevsquincy** [comment]
Bar comment, 👍

File: shared/components/brexit-secondary-results/_brexit-secondary-results.scss (hunk 3 above)
Comment (sumeetadur) [inline shared/components/brexit-secondary-results/_brexit-secondary-results.scss]: No it doesn't need to be, I just thought it was tidier to use the `&` functionality. I'll rewrite it :)

Comment (ironsidevsquincy) [inline shared/components/brexit-secondary-results/_brexit-secondary-results.scss]: Agreed, It is neater, but makes doing a search for `brexit-secondary-results__content` impossible

**Pull request merged**